app/serializers/topic_view_details_serializer.rb

```diff
@@ -95,7 +97,7 @@ def include_can_move_posts?
     scope.can_move_posts?(object.topic)
   end
 
-  def include_can_edit?
+  def can_edit
```

Comment:
I suggest we include a comment here to prevent Robin in the future from converting it to `include` again. We should note why this one needs to be explicit.